docs/reference/settings/security-settings.asciidoc

```diff
@@ -1505,6 +1505,8 @@ to establish trust.
 This diagnostic message contains information that can be used to determine the
 cause of the failure and assist with resolving the problem.
 Set to `false` to disable these messages.
+NOTE: This defaults to `false` and cannot be enabled when
+`xpack.security.fips_mode.enabled` is `true`.
```

Comment:
Based on agreement to allow FIPS users to opt-in, let's remove the 2nd part of this note.